builtin/credential/cert/path_login.go

```diff
@@ -79,14 +80,14 @@ func (b *backend) pathLogin(
 
 	clientCerts := req.Connection.ConnState.PeerCertificates
 	if len(clientCerts) == 0 {
-		return logical.ErrorResponse("no client certificate found"), nil
+		return nil, fmt.Errorf("no client certificate found")
```

Comment:
Why change this? This is more likely to be user error than an internal error.

Reply:
I'll revert this. Changed https://github.com/hashicorp/vault/pull/3642/files#diff-0f3ca593a1923ab0963f7b2f7e165b17R154 to be `logical.ErrorResponse`